src/components/Library.js

```diff
@@ -0,0 +1,72 @@
+// // App.js
```

Comment:
i think if you were to have firebase or some kind of connection for the booking data to be kept, the connection would have to be in a child component of App.js because it would involve state with every booking of the suites and state shouldn't happen in App.js or it'' rerender the whole thang. - just something to consider when you get to that bridge :D